### PR TITLE
feat: add RaspiBlitz

### PR DESCRIPTION
**Pull request by secondl1ght**
New PR for #683 due to merge conflicts.

Closes #683 #673

### CODE REVIEW

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+});
+
+export default function ConnectRaspiBlitz() {
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState(initialFormData);
+  const [loading, setLoading] = useState(false);
+
+  function handleUrl(event: React.ChangeEvent<HTMLInputElement>) {
+    let url = event.target.value.trim();
+    if (event.target.value.substring(0, 4) !== "http") {
```
Comment (bumi) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:20]: you should use `url` here, too.

**Review by bumi — CHANGES_REQUESTED** [review]
I did not look in detail, but I think this is not following the new connector screen refactorings: 

https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/connectors/ConnectMyNode/index.tsx#L91-L92

**Comment by secondl1ght** [comment]
> I did not look in detail, but I think this is not following the new connector screen refactorings:
> 
> https://github.com/getAlby/lightning-browser-extension/blob/master/src/app/screens/connectors/ConnectMyNode/index.tsx#L91-L92

Ok I updated this to use the new `ConnectorForm` component. Thanks for pointing that out I didn't notice we had a new component for this.

**Review by bumi — COMMENTED** [review]
ok, great. did you test it? did you also try to type the URL manually? 

@MoritzKa can you test again?

**Comment by secondl1ght** [comment]
> ok, great. did you test it? did you also try to type the URL manually?
> 
> @MoritzKa can you test again?

Yep I tested it and was able to connect an LND node coming from Polar, I tried typing `http` and leaving it empty and it worked both times.

**Comment by bumi** [comment]
great,thanks!

**Comment by MoritzKa** [comment]
ACK 
https://onion_address works
onion_address works
http://onion_address does not work

**Comment by bumi** [comment]
what does that mean? 
should the http:// address work? what means it does not work? what's the error?

**Comment by MoritzKa** [comment]
for me `http://` does not need to work in order to merge this PR. 
I am reporting this because sendl1ght mention above that he "tried typing `http` and leaving it empty and it worked both times." I guess he meant `https`
I think the error message was "you are trying to call a HTTPS URI from a HTTP client"

**Review by bumi — APPROVED** [review]
utACK

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import utils from "../../../../common/lib/utils";
+import TextField from "../../../components/form/TextField";
+import CompanionDownloadInfo from "../../../components/CompanionDownloadInfo";
+import ConnectorForm from "../../../components/ConnectorForm";
```
Comment (dylancom) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:6]:
```suggestion
import ConnectorForm from "@components/ConnectorForm";
```

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import utils from "../../../../common/lib/utils";
+import TextField from "../../../components/form/TextField";
+import CompanionDownloadInfo from "../../../components/CompanionDownloadInfo";
```
Comment (dylancom) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:5]:
```suggestion
import CompanionDownloadInfo from "@components/CompanionDownloadInfo";
```

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 4]
```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import utils from "../../../../common/lib/utils";
+import TextField from "../../../components/form/TextField";
```
Comment (dylancom) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:4]: import TextField from "@components/form/TextField";

File: src/app/screens/connectors/ConnectRaspiBlitz/index.tsx [hunk 5]
```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import utils from "../../../../common/lib/utils";
```
Comment (dylancom) [inline src/app/screens/connectors/ConnectRaspiBlitz/index.tsx:3]:
```suggestion
import utils from "~/common/lib/utils";
```

**Comment by bumi** [comment]
@secondl1ght can you make a PR with those updated import paths?

**Comment by secondl1ght** [comment]
Ok @dylancom is this a new way we are doing imports?

**Comment by bumi** [comment]
> Ok @dylancom is this a new way we are doing imports?

https://github.com/getAlby/lightning-browser-extension/pull/798

**Pull request merged**